### PR TITLE
8.113.0.210, snap build

### PR DESCRIPTION
**Pull request by faveoled**
logging in, camera, test call passed fine

### CODE REVIEW

**Comment by faveoled** [comment]
Help needed

File: com.skype.Client.appdata.xml [hunk 1]
```diff
@@ -18,10 +18,10 @@
     </screenshot>
```
Comment (bbhtt) [inline com.skype.Client.appdata.xml:18]: Change `<screenshot>` to `<screenshot type="default">` , github doesn't let me comment there.

**Comment by FakeShemp** [comment]
I can't test right now. If someone can confirm it works I can merge.

**Comment by 1t5j0y** [comment]
```
$ flatpak install --user https://dl.flathub.org/build-repo/86818/com.skype.Client.flatpakref
error: Can't load uri https://dl.flathub.org/build-repo/86818/com.skype.Client.flatpakref: Server returned status 404
```
Is there something I need to setup to be able to test?

**Comment by faveoled** [comment]
bot, build

**Comment by faveoled** [comment]
@1t5j0y try new build

**Comment by 1t5j0y** [comment]
Stuck on sign in screen on clicking the 'Sign in or create' button with the error message: "Unable to sign in, please check your internet connection and try again." Looks like some SSL errors.

Logs from `~/.var/app/com.skype.Client/config/logs/skype-startup.log` below.

image


<details>

<summary>Startup logs</summary>

```
Gtk-Message: 17:31:30.942: Failed to load module "canberra-gtk-module"
Gtk-Message: 17:31:30.943: Failed to load module "canberra-gtk-module"
[2:0313/173131.437368:ERROR:cert_verify_proc_builtin.cc(681)] CertVerifyProcBuiltin for redirector.gvt1.com failed:
----- Certificate i=3 (CN=GlobalSign Root CA,OU=Root CA,O=GlobalSign nv-sa,C=BE) -----
ERROR: No matching issuer found


[58:0313/173131.437676:ERROR:ssl_client_socket_impl.cc(999)] handshake failed; returned -1, SSL error code 1, net_error -202
[2:0313/173131.831469:ERROR:object_proxy.cc(623)] Failed to call method: org.freedesktop.ScreenSaver.GetActive: object_path= /org/freedesktop/ScreenSaver: org.freedesktop.DBus.Error.NotSupported: This method is not implemented
[2:0313/173132.688210:ERROR:cert_verify_proc_builtin.cc(681)] CertVerifyProcBuiltin for config.edge.skype.com failed:
----- Certificate i=2 (CN=DigiCert Global Root G2,OU=www.digicert.com,O=DigiCert Inc,C=US) -----
ERROR: No matching issuer found


[58:0313/173132.688674:ERROR:ssl_client_socket_impl.cc(999)] handshake failed; returned -1, SSL error code 1, net_error -202
[2:0313/173132.718969:ERROR:cert_verify_proc_builtin.cc(681)] CertVerifyProcBuiltin for eagle.sapphire.microsoftapp.net failed:
----- Certificate i=2 (CN=DigiCert Global Root G2,OU=www.digicert.com,O=DigiCert Inc,C=US) -----
ERROR: No matching issuer found


[58:0313/173132.719441:ERROR:ssl_client_socket_impl.cc(999)] handshake failed; returned -1, SSL error code 1, net_error -202
[2:0313/173132.720486:ERROR:cert_verify_proc_builtin.cc(681)] CertVerifyProcBuiltin for gateway.bingviz.microsoftapp.net failed:
----- Certificate i=2 (CN=DigiCert Global Root G2,OU=www.digicert.com,O=DigiCert Inc,C=US) -----
ERROR: No matching issuer found


[58:0313/173132.720771:ERROR:ssl_client_socket_impl.cc(999)] handshake failed; returned -1, SSL error code 1, net_error -202
[58:0313/173132.812908:ERROR:ssl_client_socket_impl.cc(999)] handshake failed; returned -1, SSL error code 1, net_error -202
Error occurred in handler for 'keychain:get-password': Error: No stored credentials fetched from keytar.
    at /app/extra/skypeforlinux/resources/app.asar/main.js:2:524095
    at async node:electron/js2c/browser_init:193:551
Error occurred in handler for 'keychain:get-password': Error: No stored credentials fetched from keytar.
    at /app/extra/skypeforlinux/resources/app.asar/main.js:2:524095
    at async node:electron/js2c/browser_init:193:551
Error occurred in handler for 'keychain:get-password': Error: No stored credentials fetched from keytar.
    at /app/extra/skypeforlinux/resources/app.asar/main.js:2:524095
    at async node:electron/js2c/browser_init:193:551
[2:0313/173133.871197:ERROR:cert_verify_proc_builtin.cc(681)] CertVerifyProcBuiltin for config.edge.skype.com failed:
----- Certificate i=2 (CN=DigiCert Global Root G2,OU=www.digicert.com,O=DigiCert Inc,C=US) -----
ERROR: No matching issuer found


[58:0313/173133.871514:ERROR:ssl_client_socket_impl.cc(999)] handshake failed; returned -1, SSL error code 1, net_error -202
[2:0313/173134.011331:ERROR:cert_verify_proc_builtin.cc(681)] CertVerifyProcBuiltin for redirector.gvt1.com failed:
----- Certificate i=3 (CN=GlobalSign Root CA,OU=Root CA,O=GlobalSign nv-sa,C=BE) -----
ERROR: No matching issuer found


[58:0313/173134.011652:ERROR:ssl_client_socket_impl.cc(999)] handshake failed; returned -1, SSL error code 1, net_error -202
[2:0313/173134.114396:ERROR:cert_verify_proc_builtin.cc(681)] CertVerifyProcBuiltin for login.live.com failed:
----- Certificate i=2 (CN=DigiCert Global Root CA,OU=www.digicert.com,O=DigiCert Inc,C=US) -----
ERROR: No matching issuer found


[58:0313/173134.115492:ERROR:ssl_client_socket_impl.cc(999)] handshake failed; returned -1, SSL error code 1, net_error -202
(node:2) electron: Failed to load URL: https://login.live.com/oauth20_authorize.srf?app_version=8.113.0.210&client_flight=ReservedFlight33,suhs&client_id=00000000480BC46C&cobrandid=6e63daac-8dfe-43f6-b70e-deacb69a89d6&display=touch&fl=phone2&lw=1&mkt=en-US&nopa=0&psi=skype&uaid=7ca15b5a9de744f3b3a41664ae4d340e&coa=1&scope=service::lw.skype.com::MBI_SSL&response_type=token&redirect_uri=https%3A%2F%2Flogin.live.com%2Foauth20_desktop.srf with error: ERR_CERT_AUTHORITY_INVALID
(Use `skypeforlinux --trace-warnings ...` to show where the warning was created)
Error occurred in handler for 'GUEST_VIEW_MANAGER_CALL': Error: ERR_CERT_AUTHORITY_INVALID (-202) loading 'https://login.live.com/oauth20_authorize.srf?app_version=8.113.0.210&client_flight=ReservedFlight33,suhs&client_id=00000000480BC46C&cobrandid=6e63daac-8dfe-43f6-b70e-deacb69a89d6&display=touch&fl=phone2&lw=1&mkt=en-US&nopa=0&psi=skype&uaid=7ca15b5a9de744f3b3a41664ae4d340e&coa=1&scope=service::lw.skype.com::MBI_SSL&response_type=token&redirect_uri=https%3A%2F%2Flogin.live.com%2Foauth20_desktop.srf'
    at rejectAndCleanup (node:electron/js2c/browser_init:165:7810)
    at EventEmitter.failListener (node:electron/js2c/browser_init:165:8023)
    at EventEmitter.emit (node:events:538:35) {
  errno: -202,
  code: 'ERR_CERT_AUTHORITY_INVALID',
  url: 'https://login.live.com/oauth20_authorize.srf?app_version=8.113.0.210&client_flight=ReservedFlight33,suhs&client_id=00000000480BC46C&cobrandid=6e63daac-8dfe-43f6-b70e-deacb69a89d6&display=touch&fl=phone2&lw=1&mkt=en-US&nopa=0&psi=skype&uaid=7ca15b5a9de744f3b3a41664ae4d340e&coa=1&scope=service::lw.skype.com::MBI_SSL&response_type=token&redirect_uri=https%3A%2F%2Flogin.live.com%2Foauth20_desktop.srf'
}
```

</details>

**Comment by FakeShemp** [comment]
bot, build

**Pull request merged**